src/components/vedtak-liste/vedtak-liste-tekster.ts

```diff
@@ -2,7 +2,8 @@ const VedtakListeTekster = {
     'vedtak-liste.sidetittel': 'Svar på søknader',
     'vedtak-liste.ingen-nye-soknader': 'Du har ingen nye svar på søknader fra NAV.',
     'vedtak-liste.ingen-tidligere-soknader': 'Du har ingen tidligere svar på søknader fra NAV.',
-    'vedtak-liste.sorter.etter': 'Sorter etter'
+    'vedtak-liste.sorter.etter': 'Sorter etter',
+    'vedtak-liste.lenke-arkiverte-vedtak': 'Se alle tidliger svar på sykepengesøknader'
```

Comment:
Se alle tidligere

Reply:
Se alle Svar på søknader om sykepenger på Ditt NAV

Reply:
https://trello.com/c/LXmQKKiG/1604-som-spinnsyn-frontend-vil-jeg-ha-en-lenke-til-dokumentoversikt-p%C3%A5-ditt-nav-for-%C3%A5-kunne-se-arkiverte-vedtak